<ui version="4.0" >
 <author></author>
 <comment>*********************************************************************
**
** Copyright (C) 1992-$THISYEAR$ Trolltech AS. All rights reserved.
**
** This file is part of the $MODULE$ of the Qt Toolkit.
**
** $LICENSE$
**
** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
**
*********************************************************************</comment>
 <exportmacro></exportmacro>
 <class>TableWidgetEditor</class>
 <widget class="QDialog" name="TableWidgetEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit Tree Widget</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="1" >
    <widget class="QGroupBox" name="rowsBox" >
     <property name="title" >
      <string>Rows</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QListWidget" name="rowsListWidget" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>13</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::TabFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QToolButton" name="newRowButton" >
          <property name="text" >
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="renameRowButton" >
          <property name="text" >
           <string>Rename</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deleteRowButton" >
          <property name="text" >
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Ignored</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="moveRowUpButton" >
          <property name="text" >
           <string>U</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="pixmapRowLabel" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>0</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Pixmap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="previewPixmapRowButton" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deletePixmapRowButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Ignored</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="moveRowDownButton" >
          <property name="text" >
           <string>D</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="columnsBox" >
     <property name="title" >
      <string>Columns</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QListWidget" name="columnsListWidget" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>13</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::TabFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QToolButton" name="newColumnButton" >
          <property name="text" >
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="renameColumnButton" >
          <property name="text" >
           <string>Rename</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deleteColumnButton" >
          <property name="text" >
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Ignored</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="moveColumnUpButton" >
          <property name="text" >
           <string>U</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="pixmapColumnLabel" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>0</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Pixmap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="previewPixmapColumnButton" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deletePixmapColumnButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Ignored</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="moveColumnDownButton" >
          <property name="text" >
           <string>D</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="2" row="0" column="0" >
    <widget class="QGroupBox" name="itemsBox" >
     <property name="title" >
      <string>Table Items</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QFrame" name="currentItemBox" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>&amp;Text</string>
           </property>
           <property name="buddy" >
            <cstring>itemTextLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="itemTextLineEdit" />
         </item>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>&amp;Pixmap</string>
           </property>
           <property name="buddy" >
            <cstring>previewPixmapItemButton</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="previewPixmapItemButton" >
           <property name="text" >
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="deletePixmapItemButton" >
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QTableWidget" name="tableWidget" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>431</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>columnsListWidget</tabstop>
  <tabstop>newColumnButton</tabstop>
  <tabstop>renameColumnButton</tabstop>
  <tabstop>deleteColumnButton</tabstop>
  <tabstop>moveColumnUpButton</tabstop>
  <tabstop>previewPixmapColumnButton</tabstop>
  <tabstop>deletePixmapColumnButton</tabstop>
  <tabstop>moveColumnDownButton</tabstop>
  <tabstop>itemTextLineEdit</tabstop>
  <tabstop>previewPixmapItemButton</tabstop>
  <tabstop>deletePixmapItemButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>TableWidgetEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>342</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel" >
     <x>159</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>TableWidgetEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>437</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel" >
     <x>178</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
